drawee/src/main/java/com/facebook/drawee/view/GenericDraweeView.java

```diff
@@ -60,6 +61,8 @@
  */
 public class GenericDraweeView extends DraweeView<GenericDraweeHierarchy> {
 
+  protected String mImageUri= "";
```

Comment:
Uri doesn't really belong to `GenericDraweeView`. This would break separation of concerns. This should be moved to `SimpleDraweeView`, but even there, we should not keep it as a member variable. Someone can later set an ImageRequest and we would end up with a weird state where mUri points to one image whereas the actual ImageRequest used points to another..